NeoMathEngine/src/CPU/CpuMathEngine.h

```diff
@@ -629,7 +629,7 @@ class CCpuMathEngine : public IMathEngine, public IRawMemoryManager {
 	void RowwiseExecute( const CBlobDesc& inputDesc, CRowwiseOperationDesc** operations, int operationCount,
 		const CFloatHandle& input, const CFloatHandle& output ) override;
 
-	IPerformanceCounters* CreatePerformanceCounters() const override;
+	IPerformanceCounters* CreatePerformanceCounters( bool isOnlyTime = false ) const override;
```

Comment:
This header isn't public. No need for default value for `isOnlyTime`. It's needed only in public interface (`IMathEngine`)